main.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from database import SessionLocal
+import crud
+import schemas
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/author/", response_model=list[schemas.Author])
+def get_author(db: Session = Depends(get_db)):
+    return crud.get_all_authors(db)
```

Comment:
The `get_author` endpoint does not implement pagination. Consider adding `skip` and `limit` query parameters to enable pagination, as required by the task description.